unity-renderer/Assets/DCLServices/WearablesCatalogService/WearablesCatalogService.cs

```diff
@@ -0,0 +1,97 @@
+            this.webInterfaceWearablesCatalogService = webInterfaceWearablesCatalogService;
+            this.wearablesCatalog = wearablesCatalog;
+            this.kernelConfig = kernelConfig;
+        }
+
+        public void Initialize()
+        {
+            kernelConfig.EnsureConfigInitialized().Then(config => OnKernelConfigChanged(config, null));
+            kernelConfig.OnChange += OnKernelConfigChanged;
+        }
+
+        public void Dispose()
+        {
+            if (kernelConfig != null)
+                kernelConfig.OnChange -= OnKernelConfigChanged;
+
+            wearablesCatalogServiceInUse?.Dispose();
+        }
+
+        public UniTask<WearableItem[]> RequestOwnedWearablesAsync(string userId, int pageNumber, int pageSize, CancellationToken ct) =>
+            wearablesCatalogServiceInUse.RequestOwnedWearablesAsync(userId, pageNumber, pageSize, ct);
+
+        public UniTask<WearableItem[]> RequestBaseWearablesAsync(CancellationToken ct) =>
+            wearablesCatalogServiceInUse.RequestBaseWearablesAsync(ct);
+
+        public UniTask<WearableItem[]> RequestThirdPartyWearablesByCollectionAsync(string userId, string collectionId, CancellationToken ct) =>
+            wearablesCatalogServiceInUse.RequestThirdPartyWearablesByCollectionAsync(userId, collectionId, ct);
+
+        public UniTask<WearableItem[]> RequestWearablesAsync(string[] wearableIds, CancellationToken ct) =>
+            wearablesCatalogServiceInUse.RequestWearablesAsync(wearableIds, ct);
+
+        public UniTask<WearableItem> RequestWearableAsync(string wearableId, CancellationToken ct) =>
+            wearablesCatalogServiceInUse.RequestWearableAsync(wearableId, ct);
+
+        public void AddWearablesToCatalog(WearableItem[] wearableItems) =>
+            wearablesCatalogServiceInUse.AddWearablesToCatalog(wearableItems);
+
+        public void RemoveWearablesFromCatalog(IEnumerable<string> wearableIds) =>
+            wearablesCatalogServiceInUse.RemoveWearablesFromCatalog(wearableIds);
+
+        public void RemoveWearablesInUse(IEnumerable<string> wearablesInUseToRemove) =>
+            wearablesCatalogServiceInUse.RemoveWearablesInUse(wearablesInUseToRemove);
+
+        public void EmbedWearables(IEnumerable<WearableItem> wearables) =>
+            wearablesCatalogServiceInUse.EmbedWearables(wearables);
+
+        public void Clear() =>
+            wearablesCatalogServiceInUse.Clear();
+
+        private void OnKernelConfigChanged(KernelConfigModel currentKernelConfig, KernelConfigModel previous)
+        {
+            kernelConfig.OnChange -= OnKernelConfigChanged;
+
+            if (currentKernelConfig.urlParamsForWearablesDebug)
+            {
+                WebInterfaceWearablesCatalogService.Instance.Initialize(new WearablesWebInterfaceBridge(), wearablesCatalog);
+                wearablesCatalogServiceInUse = webInterfaceWearablesCatalogService;
+                lambdasWearablesCatalogService?.Dispose();
```

Comment:
It's true that I don't need to dispose `lambdasWearablesCatalogService` in this case (so I have removed it), but I need to dispose the `webInterfaceWearablesCatalogService` in the `else` because I want to destroy its associated component that was instantiated in the scene (in the `Main` game object) to be able to receive messages from kernel. I decided to put this `destroy` operation inside its own  `Dispose` method.